src/addons/AddonExhaustibleWater.h

```diff
@@ -19,19 +19,23 @@
 
 #pragma once
 
-#include "AddonBool.h"
+#include "AddonList.h"
 #include "mygettext/mygettext.h"
 
 /**
  *  Addon for allowing to have unlimited resources.
  */
-class AddonExhaustibleWells : public AddonBool
+class AddonExhaustibleWater : public AddonList
 {
 public:
-    AddonExhaustibleWells()
-        : AddonBool(AddonId::EXHAUSTIBLE_WELLS, ADDONGROUP_ECONOMY, _("Exhaustible Wells"),
-                    _("Wells will now dry out, limiting the available water"), 0)
-    {}
+    AddonExhaustibleWater()
+        : AddonList(AddonId::EXHAUSTIBLE_WATER, ADDONGROUP_ECONOMY, _("Exhaustible Water"),
+                    _("If Water is exhaustible wells will now dry out."), 0)
```

Comment:
Maybe say also that "water everywhere" disables water found notifications

Reply:
Added the hint to the Addon-Description.